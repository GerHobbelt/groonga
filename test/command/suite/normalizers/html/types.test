# U+3042 HIRAGANA LETTER A
normalize \
  'NormalizerHTML' \
  "a&lt;b&#x80;c&#x3042;d" \
  WITH_TYPES
