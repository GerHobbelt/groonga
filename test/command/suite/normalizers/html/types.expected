normalize   'NormalizerHTML'   "a&lt;b&#x80;c&#x3042;d"   WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "a<b€cあd",
    "types": [
      "alpha",
      "symbol",
      "alpha",
      "symbol",
      "alpha",
      "hiragana",
      "alpha"
    ],
    "checks": [

    ]
  }
]
